hyperspy/_signals/eds.py

```diff
@@ -180,15 +180,19 @@ def sum(self, axis=None):
         if axis is None:
             axis = self.axes_manager.navigation_axes
         # modify time spend per spectrum
-        s = super(EDSSpectrum, self).sum(axis)
+        s = super(EDSSpectrum, self).sum(axis, out)
```

Comment:
While not required, I think it's more elegant to pass the arguments not as positional, but as keyword arguments - especially because they are defined like so. Then we get the freedom of (accidentally) switching the order, either while using or in the definition